app/controllers/hosts_controller.rb

```diff
@@ -546,7 +546,8 @@ def process_taxonomy
   end
 
   def template_used
-    host      = Host.new(clean_interfaces_attributes.except(:host_parameters_attributes))
+    host = params[:id] ? Host.find(params[:id]) : Host.new
```

Comment:
Modified to filter all *_ids paramteres and added extra comment about it.

> Would using .readonly help?

No :cry:, readonly seems to be effective only on the object itself, associations ignore it.